src/inc_gauged_raz.rs

```diff
@@ -248,9 +248,9 @@ impl<E: Debug+Clone+Eq+Hash+'static, M:RazMeta<E>> RazTree<E,M> {
 				tree.fold_lr_meta(start_name,init,Rc::new(move|a,e,l,n|{
 					match e {
 						TreeData::Leaf(ref vec) => {
-							ns(name_of_string(String::from("extra name")),||{
+							//ns(name_of_string(String::from("extra name")),||{
 								finbin(vec.iter().fold(a,|a,e|{bin(a,e)}),n)
```

Comment:
I think this needs to be in a namespace, since the name is used twice. Your particular test ignores it. Or I can remove the other name from the test. That would probably be better. Let's see if I can update the pull request...